bus/bus.go

```diff
@@ -615,9 +615,9 @@ func NewServer(b *Bus) http.Handler {
 		"GET    /contractsets/:name/contracts": b.contractSetContractsHandler,
 		"PUT    /contractsets/:name":           b.contractSetsNameHandlerPUT,
 
-		"GET    /objects/*key":               b.objectsKeyHandlerGET,
-		"PUT    /objects/*key":               b.objectsKeyHandlerPUT,
-		"DELETE /objects/*key":               b.objectsKeyHandlerDELETE,
+		"GET    /objects/store/*key":         b.objectsStoreKeyHandlerGET,
+		"PUT    /objects/store/*key":         b.objectsStoreKeyHandlerPUT,
```

Comment:
`PUT` and `DELETE` currently don't handle trailing slashes in the `*key`, they should probably throw a 400